resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -173,7 +173,7 @@ function task_install {
     ln -s "../apps/${real_name}/bin/<%= basename %>" "${DESTDIR}${symbindir}/<%= basename %>"
     ln -s "../server/apps/${real_name}/bin/<%= basename %>" "${DESTDIR}${uxbindir}/<%= basename %>"
 <% end -%>
-    install -d -m 0755 "${DESTDIR}${rundir}"
+    install -d -m 0755 -o <%= EZBake::Config[:user] %> -g <%= EZBake::Config[:group] %> "${DESTDIR}${rundir}"
```

Comment:
I think if we take this out though that, at least on Debian, we'll need to have the logic that creates/chowns the directory in the init script be able to chown the directory if it already exists - i.e., if it was installed with root ownership.  In that case, going with {{/usr/bin/install}} would seem like a better approach.

I put up an alternative PR for this, #376, which switches the mkdir/chown pattern over to {{/usr/bin/install}} in all places.  It also adds in some extra error handling for the start script which should cause the service to fail right away in the event that permissions on the directory cannot be corrected before the process is started.  I'll do some testing with that across a few OSes to see that it does what I expect it to do.

If there's an alternate way to get the run dir built into the package with the proper USER/GROUP for Debian like we apparently have for RedHat, I'm definitely open to adding that in as well.  Hopefully all of the other changes with whatever we merge will compensate for the lack of that in any event.